inc/renderer.php

```diff
@@ -34,16 +34,19 @@ public function add_inline_css( $post_id, $css ) {
 	 * Generate the CSS for the page layout.
 	 *
 	 * @param $post_id
-	 * @param $layout_data
 	 * @param $panels_data
+	 * @param $layout_data
 	 *
 	 * @return string
 	 */
-	private function generate_css( $post_id, $layout_data, $panels_data ) {
+	public function generate_css( $post_id, $panels_data, $layout_data) {
 		// Exit if we don't have panels data
-		if ( empty( $layout_data ) ) {
+		if ( empty( $layout_data ) && empty( $panels_data ) ) {
 			return;
 		}
+		if( empty( $layout_data ) ) {
```

Comment:
Just a personal preference thing, but this could probably be combined with the above conditional. Something like 
```
if ( empty( $layout_data ) ) {
    if ( empty( $panels_data ) ) {
        return;
    }
    $layout_data = $this->get_panels_layout_data( $panels_data );
}